Create a new doc page for Main UI‘s special features (#2188)
openhab/openhab-docs: ui/mainui.md

To setup an Item as the UI command Item on your device, open the "Help & About" page from the menu and select an Item for "Listen for UI command to".

The command Item has to be a `String` Item.
The UI is listening for commands to that Item, and if an Item command is a valid command string, the UI executes the given command.

Currently, the supported commands are:

- `navigate:$path`:
  Navigate to a given `$path`, e.g. `navigate:/locations` to navigate to the "Locations" tab of the home page or `navigate:/page/my_custom_page` to navigate to a page with the ID `my_custom_page`.
- `popup:$target`, `popover:$target` and `sheet:$target`:
  Open a `$target`, which can be a page (`$target` is `page:` + a page ID), a custom widget (`$target` is `widget:` + widget ID) or any [`oh-` component]({{base}}/ui/components/) (`$target` is the component name), in a popup, popover or sheet.
  
  For example, to open `oh-clock-card` inside a popup, send `popup:oh-clock-card` to the UI command Item.

> - `notification:$text:$title:$subtitle:$titleRight:$closeTimeout`:

Reviewer: Then would you mind adding this as an example or mention this?

> - `notification:$text:$title:$subtitle:$titleRight:$closeTimeout`:

Author: I’ve added this sentence here:

> To leave out a parameter, leave a colon space empty, e.g. `notification:$text:$title:::$closeTimeout`.

Should I still add an example?

> - `notification:$text:$title:$subtitle:$titleRight:$closeTimeout`:

Reviewer: No, that's fine.

  Display a notification inside the UI:
  The `$text` parameter is mandatory, all other parameters are optional.
  `$closeTimeout` is in milliseconds and defaults 5000, which means by default a notification will be closed after 5 seconds.
  
  For example, send `notification:This is the text.:This is the title.:This is the subtitle.:This is the right title.` to the UI command Item to display the following notification:
  <p align="center"><img style="max-width: 40%;" src="images/notification.png"/></p>
- `close`:
  Close all popups, popovers and sheets.
- `back`:

>   Naviate back.

Reviewer: Navigate

The developer sidebar search ignores case (so it doesn't matter whether you search `LivingRoom_Light` or `livingroom_light`) and searches inside the following entities:

- Things
  - UID
  - label
- Items
  - name
  - label
  - tags (requires exact match, i.e. if the Item has a tag `Lightbulb`, only the search query `Lightbulb` (case doesn't matter) will match)
  - metadata
- Pages (including Sitemaps)
  - uid
  - label
  - content (widgets, tabs, charts, etc. with their configuration)/slots
- Transformations
  - uid
  - label
  - type
- Rules, Scenes & Scripts
  - Item name & Thing UID of triggers, actions & conditions
  - script code (e.g. Rules DSL, JavaScript, and even Blockly)
  - script MIME types (requires exact match)
  - Blockly scripts (using `block`, `blockly` or `blocksource` as search string, where case is ignored)
  - tags (requires exact match)
- Persistence Configurations:
  - label & service id of persistence service
  - Items persisted by persistence service

## Web Audio Sink

> Main UI implements a web audio sink, which means the openHAB server can play audio through on a client device through the browser.

Reviewer 2: ```suggestion
Main UI implements a web audio sink which allows the openHAB server to play audio through the web browser.
```
